compiler/dyno/test/common.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "chpl/types/all-types.h"
+#include "chpl/uast/Identifier.h"
+#include "chpl/uast/Module.h"
+#include "chpl/uast/Record.h"
+#include "chpl/uast/Variable.h"
+
+namespace impl {
```

Comment:
Why do we have this `namespace impl` ?

Reply:
I didn't want including the header to dump a bunch of things into the source file's scope / namespace. So `impl` "surrounds" the various `using namespace` directives (which themselves make the header definition much cleaner) and prevents them from leaking into files that include `common.h`

Reply:
But you have `using namespace impl;` - doesn't that transitively get the other `using namespace` directives? Anyway for the test code I don't think it's a big deal if they automatically get a bunch of `using namespace` from `common.h`.

Reply:
I thought it wouldn't, but looks like I thought wrong. I'll get rid of the `impl` namespace since a proper solution doesn't seem worth it at the time.